libs/java/auth_core/src/main/java/com/yahoo/athenz/auth/token/jwts/JwtsSigningKeyResolver.java

```diff
@@ -45,6 +45,9 @@ public class JwtsSigningKeyResolver implements SigningKeyResolver {
     private static final ObjectMapper JSON_MAPPER = initJsonMapper();
     private final SSLContext sslContext;
     private final String jwksUri;
+    private final boolean skipConfig;
```

Comment:
any reason why we're not storing this field if we're not using it anywhere else?

Reply:
Thank you, as you mentioned, since we're no longer using this field from anywhere else, we decided to remove it.